rll/setup-automatic-upgrade.sh

```diff
@@ -18,23 +18,29 @@
 # ...
 #
 
-# Add entry in /etc/cron.d/ to check and execute an upgrade for rightlink daily.
+# Determine base directory of rightlink / rsc
+[[ -e /usr/local/bin/rightlink ]] && bin_dir=/usr/local/bin || bin_dir=/opt/bin
```

Comment:
We never use the rightlink binary in this file.  To make it simple, let's check for `/usr/local/bin/rsc` instead.